src/MuxManager.cpp

```diff
@@ -570,6 +570,9 @@ void MuxManager::handleWarmRestartReconciliationTimeout(const boost::system::err
         MUXLOGWARNING("Reconciliation timed out after warm restart, set service to reconciled now.");
     }
 
+    uint32_t rc = system("sudo config muxcable mode auto all"); // rc > 0 means success
```

Comment:
Do you not want to handle rc, at least for NOTICE vs WARNING logs? If config to auto fails, do you still want to conclude to reconciled state?

Reply:
> Do you not want to handle rc, at least for NOTICE vs WARNING logs? If config to auto fails, do you still want to conclude to reconciled state?

What will happen if linkmgrd never enter "reconciled" state?

Reply:
The finalizer script waits on linkmgrd to reconcile: https://github.com/sonic-net/sonic-buildimage/blob/7cd9f132535a90c2080cccfc418b976e471f69b5/files/image_config/warmboot-finalizer/finalize-warmboot.sh#L13C31-L13C31

I can't think of a way to fix this failed to reconcile state. The best I can think of is if the reconciliation did not happen within 5 minutes then we consider warmboot failed, and a log is generated from finalizer.
There is no alert generated today (AFAIK), but I think we should generate alert when such a log is created.

Reply:
I think a warning on failure woud be nice.